examples/advanced_agents/sandbox_agent/src/nat_sandbox_agent/sandbox/daytona_sandbox.py

```
@@ -0,0 +1,225 @@
+
+    async def read_file(self, path: str) -> str:
+        """Read file content from the Daytona sandbox."""
+        if not self._sandbox:
+            raise RuntimeError("Sandbox not started")
+
+        try:
+            # Wrap sync call to avoid blocking
+            content = await asyncio.get_running_loop().run_in_executor(
+                None, lambda: self._sandbox.fs.read_file(path)
+            )
+            return content
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)